en/docs/getting-started/basic-architecture.md

```diff
@@ -1,73 +1,72 @@
-#Basic Architecture and Key Components
+# Basic Architecture and Key Components
 
-![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)
+[![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)]({{base_path}}/assets/img/get_started/basic-architecture.png)
```

Comment:
Why are there diagonal lines in the background of this image?
<img width="1096" alt="Screenshot 2020-12-09 at 17 22 08" src="https://user-images.githubusercontent.com/5195851/101627360-44b79080-3a44-11eb-9fe8-c137df6d57cd.png">